installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -14,7 +14,8 @@ In {product-title} version {product-version}, you can install a customized clust
 xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
 processes.
 
-* Have access to a {rh-openstack} administrator's account
+* Have a storage service installed in {rh-openstack}, like Object Block Storage (Cinder) or Object Storage (Swift)
```

Comment:
The `Object Block Storage (Cinder)` part is a typo, right? It should be `Block Storage (Cinder)`.

Reply:
@mandre Whoops. Yeah. Thanks!